my_price_scout/tests/test_product.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+
+
+from files.product import Product
+from files.specific_product import Specific_Product
+
+
+def test_exists():
+    assert Product
+
+
+def test_add_new_specific_product(specific_product):
+    product = Product("ups", 180)
+    print(product)
+
+    product.add_new_specific_product(specific_product)
+
+    expected = [
+        "ups",
+        180,
+        ["amazon",
+        "https://www.amazon.com/APC-Battery-Protector-BackUPS-BX1500M/dp/B06VY6FXMM?ref_=Oct_DLandingS_D_d1d1e0d6_60&smid=ATVPDKIKX0DER&th=1"]
+    ]
+
+    assert product.product_name == expected[0]
+    assert product.target_price == expected[1]
+    assert product.specific_product_list[0].website == 'amazon'
+    assert product.specific_product_list[0].url == "https://www.amazon.com/APC-Battery-Protector-BackUPS-BX1500M/dpu/B06VY6FXMM?ref_=Oct_DLandingS_D_d1d1e0d6_60&smid=ATVPDKIKX0DER&th=1"
+    assert product.specific_product_list[0].price == 10
+
+def test_remove_old_specific_product(specific_product):
+    product = Product("ups", 180)
+    # print(product)
+
+    product.remove_old_specific_product("amazon", "https://www.amazon.com/APC-Battery-Protector-BackUPS-BX1500M/dp/B06VY6FXMM?ref_=Oct_DLandingS_D_d1d1e0d6_60&smid=ATVPDKIKX0DER&th=1")
```

Comment:
Looks like we are trying to remove something that doesn't exist here, please consider adding a specific product before removing it